docs/content/users/quickstart.md

```diff
@@ -176,18 +176,18 @@ Environment variables will be automatically added to your `.env` file to simplif
     ddev config --project-type=craftcms --docroot=web
 
     # Boot the project and install the starter project:
-    ddev start
     ddev composer create-project --no-scripts craftcms/craft
-    ddev craft install/craft \
+
+    # Install Craft CMS:
+    ddev craft install \
         --username=admin \
         --password=Password123 \
         --email=admin@example.com \
-        --site-name='$DDEV_PROJECT' \
         --language=en \
-        --site-url='$DDEV_PRIMARY_URL'
+        --site-url='$PRIMARY_SITE_URL'
 
     # Login using `admin` user and `Password123` password
-    ddev launch
+    ddev launch admin
```

Comment:
Bring them to the control panel login.

Reply:
I would normally put /admin, but it doesn't matter.